terraform/azure-ghaf-infra.tf

```diff
@@ -19,19 +26,10 @@ provider "azurerm" {
 data "sops_file" "ghaf_infra" {
   source_file = "secrets.yaml"
 }
-# Backend for storing tfstate (see ./azure-storage)
-terraform {
-  backend "azurerm" {
-    resource_group_name  = "ghaf-infra-storage"
-    storage_account_name = "ghafinfrastatestorage"
-    container_name       = "ghaf-infra-tfstate-container"
-    key                  = "ghaf-infra.tfstate"
-  }
-}
 # Resource group
 resource "azurerm_resource_group" "ghaf_infra_tf_dev" {
   name     = "ghaf-infra-tf-dev"
-  location = "swedencentral"
+  location = "northeurope"
```

Comment:
Nice, we are moving to more reliable, av zones! 

Btw also maybe would nice if we parameterize the variable:

```
variable "location" {
description = "The Azure region where resources will be created."
type = string
default = "northeurope"
}

```

Reply:
I would propose we parameterize later when we actually have a use-case that requires the resource group location (or other options) to be configurable. 

Might be my lack of Terraform experience, but right now, I don't see the benefit or use-case for adding the extra variable.

@karim20230 : feel free correct me if I'm wrong, or fail to see the use-case.

Reply:
@henrirosten  yessir  we can do it in future ..